tests/subscriptions/test_executor_consumer.py

```diff
@@ -356,6 +356,11 @@ def test_produce_result() -> None:
     strategy.poll()
     assert commit.call_count == 1
 
+    # Commit is throttled so if we immediately submit another message, the commit count will not change
+    strategy.submit(message)
+    strategy.poll()
+    assert commit.call_count == 1
```

Comment:
Can you add a strategy.join() and verify that the commit call count goes up

Reply:
Sure